### ISSUE TITLE
CP2102 USB to UART Bridge not recognized

### ISSUE THREAD

**Issue by dk7xe** [NXPmicro/meta-nxp-hovergames#13]
CP2102 USB bridge is not recognized in Navq Demo image.

Dmesg on NavQ gives the following output
```
[    8.075603] qca9377: driver loaded in 580000
[    9.278451] fec 30be0000.ethernet eth0: Link is Up - 1Gbps/Full - flow control rx/tx
[    9.278485] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[  301.829610] usb 1-1: new full-speed USB device number 2 using ci_hdrc
navq@imx8mmnavq:~$ 
```
No ttyUSB is showing up.

Below reference for same USB device on Ubuntu PC:

```
[ 8352.424344] usb 2-2: new full-speed USB device number 4 using xhci_hcd
[ 8352.573778] usb 2-2: New USB device found, idVendor=10c4, idProduct=ea60, bcdDevice= 1.00
[ 8352.573780] usb 2-2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[ 8352.573781] usb 2-2: Product: CP2102 USB to UART Bridge Controller
[ 8352.573782] usb 2-2: Manufacturer: Silicon Labs
[ 8352.573783] usb 2-2: SerialNumber: 0001
[ 8352.594781] usbcore: registered new interface driver usbserial_generic
[ 8352.594792] usbserial: USB Serial support registered for generic
[ 8352.598418] usbcore: registered new interface driver cp210x
[ 8352.598828] usbserial: USB Serial support registered for cp210x
[ 8352.599002] cp210x 2-2:1.0: cp210x converter detected
[ 8352.600710] usb 2-2: cp210x converter now attached to ttyUSB0
```

**Comment by dk7xe** [comment]
@PetervdPerk-NXP this is the usb-serial adapter of the X4 Lidar that is not recognized by the NavQ Demo image.

**Comment by dk7xe** [comment]
Should be solved by https://github.com/NXPmicro/meta-nxp-hovergames/pull/17

**Comment by PetervdPerk-NXP** [comment]
Fixed in #17
